Microsoft.DotNet.Try.Jupyter/JupyterRequestContextHandler.cs

```diff
@@ -30,155 +35,190 @@ public JupyterRequestContextHandler(PackageRegistry packageRegistry)
-                        {
-                            ExecutionCount = _executionCount
-                        };
-
-                        // send to server
-                        var executeReply = messageBuilder.CreateMessage(
-                            MessageTypeValues.ExecuteReply, 
-                            executeReplyPayload, 
-                            delivery.Command.Request.Header);
-
-                        executeReply.Identifiers = delivery.Command.Request.Identifiers;
-
-                        serverChannel.Send(executeReply);
-                    }
-                    else
-                    {
-                        var errorContent = new Error
-                        {
-                            EName = string.IsNullOrWhiteSpace(result.Exception)
-                                        ? "Compile Error" 
-                                        : "Unhandled Exception",
-                            EValue = output,
-                            Traceback = new List<string>()
-                        };
-
-                        //  reply Error
-                        var executeReplyPayload = new ExecuteReplyError(errorContent)
-                        {
-                            ExecutionCount = _executionCount
-                        };
-
-                        // send to server
-                        var executeReply = messageBuilder.CreateMessage(
-                            MessageTypeValues.ExecuteReply, 
-                            executeReplyPayload, 
-                            delivery.Command.Request.Header);
-
-                        executeReply.Identifiers = delivery.Command.Request.Identifiers;
-
-                        serverChannel.Send(executeReply);
-
-                        if (!executeRequest.Silent)
-                        {
-                            // send on io
-                            var error = messageBuilder.CreateMessage(
-                                MessageTypeValues.Error, 
-                                errorContent, 
-                                delivery.Command.Request.Header);
-                            ioPubChannel.Send(error);
-
-                            // send on stderr
-                            var stdErr = new StdErrStream
-                            {
-                                Text = errorContent.EValue
-                            };
-                            var stream = messageBuilder.CreateMessage(
-                                MessageTypeValues.Stream, 
-                                stdErr, 
-                                delivery.Command.Request.Header);
-                            ioPubChannel.Send(stream);
-                        }
-                    }
-
-                    if (!executeRequest.Silent && resultSucceeded)
+            // executeResult data
+            var executeResultData = new ExecuteResult
+            {
+                Data = new JObject
```

Comment:
Can we use a more specific type here? Mixing our own message exchange types with `NewtonSoft.Json` types is awkward.

Reply:
Data is a "python dictionary" can try to use dictionary of string,object

Reply:
Or maybe create a custom type that just inherits dictionary but is a bit more of a signpost to where this abstraction comes from?